docs/site/Debugging-with-vscode.md

```diff
@@ -0,0 +1,38 @@
+---
+lang: en
+title: 'Debugging with VS Code'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, Debug
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Debugging-with-vscode.html
+---
+
+LoopBack team develops with editor Visual Studio Code in which we defined some
+tasks to help people debug their projects.
+
+### Debugging Packages
```

Comment:
Does it refer to debugging LoopBack itself? If so, would DEVELOPING.md be a better place?

Reply:
@dhmlau I would prefer to organize all the vscode debug docs on one page. I have a section in the troubleshooting page called "## Running VS Code Tasks" that refers to this page.